test/utils/urlEncoding/decodeSymbol.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { decodeSymbol } from '../../../src/utils'
+describe('Decoding symbol', () => {
+  test('A-Z symbol', () => {
+    expect(decodeSymbol('TNK')).toBe('TNK')
+  })
+
+  test('Special chars', () => {
+    expect(decodeSymbol("Token%20%2F21%20%_%2055%3F%20**%20%7C%_%3D%2B%23%40%25%5E%26%60'%22")).toBe(
+      'Token /21 - 55? ** |-=+#@%^&`\'"',
+    )
+  })
+  test('Dash in the symbol', () => {
+    expect(decodeSymbol('TN%_HUI')).toBe('TN-HUI')
```

Comment:
Yeah baby! not pretty, but what the hell, I would be surprised if we ever see a real token with a dash. 
I think no-one would care or notice that we encode the the dash in a non standard way, so I'm all for it!

Reply:
As we've seen, browsers do not like encoding that's not really an encoding:

![screenshot_2020-04-17_09-24-39](https://user-images.githubusercontent.com/43217/79595509-d6c4f000-8093-11ea-943a-f562328c3ea5.png)